docs/source/conf.py

```diff
@@ -60,11 +60,13 @@
 ]
 
 # intersphinx mapping for outside starfish linking
-intersphinx_mapping = {'xarray': ('http://xarray.pydata.org/en/stable', None),
-                       'numpy': ('https://docs.scipy.org/doc/numpy/', None),
-                       'scipy': ('http://docs.scipy.org/doc/scipy/reference/', None),
-                       'trackpy': ('https://soft-matter.github.io/trackpy/v0.3.2/',  None),
-                       'skimage': ('https://scikit-image.org/docs/dev/', None)}
+intersphinx_mapping = {
+    'xarray': ('http://xarray.pydata.org/en/stable', None),
+    'numpy': ('https://docs.scipy.org/doc/numpy/', None),
+    'scipy': ('http://docs.scipy.org/doc/scipy/reference/', None),
+    'trackpy': ('https://soft-matter.github.io/trackpy/v0.3.2/',  None),
+    'skimage': ('https://scikit-image.org/docs/dev/', None),
+    }
```

Comment:
indent -= 4

Reply:
oops. merged as soon as I saw "approved" and before I saw the comment :(

I'll take care of these in a new PR